packages/pegasus/src/courier.js

```diff
@@ -40,6 +40,7 @@ export const getCourierPK = (key, keyToCourierPK) => {
  * @property {(zcfSeat: ZCFSeat, amounts: AmountKeywordRecord) => void} retain
  * @property {(zcfSeat: ZCFSeat, amounts: AmountKeywordRecord) => void} redeem
  * @property {ERef<TransferProtocol>} transferProtocol
+ * @property {ReturnType<import('@agoric/whenable').prepareWhenableModule>['when']} when
```

Comment:
That type is kind of a mouthful; seems like `@agoric/whenable` should export it.

Reply:
Good though here. I'll look into making this an exported type